web/client/utils/GeoStoryUtils.js

```diff
@@ -496,6 +497,78 @@ export const parseHashUrlScrollUpdate = (url, hash = '', storyId, sectionId, col
     return null;
 };
 
+// customGetEntityId is a utility function used by html-to-draftjs library in order
+// to create anhor tag chunks
+export const customGetEntityId = (node) => {
+    let entityId = undefined;
+    if (
+        node instanceof HTMLAnchorElement
+    ) {
+        const entityConfig = {};
+        if (node.dataset && node.dataset.mention !== undefined) {
+            entityConfig.url = node.href;
+            entityConfig.text = node.innerHTML;
+            entityConfig.value = node.dataset.value;
+            entityId = Entity.__create(
+                'MENTION',
+                'IMMUTABLE',
+                entityConfig,
+            );
+        } else {
+            entityConfig.url = node.getAttribute ? node.getAttribute('href') || node.href : node.href;
+            entityConfig.title = node.innerHTML;
+            entityConfig.targetOption = node.target;
+            const interactionType = node.getAttribute && node.getAttribute('data-geostory-interaction-type');
+            const interactionParams = node.getAttribute && node.getAttribute('data-geostory-interaction-params');
+
+            const attributes = {
+                type: interactionType,
+                params: interactionParams
+            };
```

Comment:
If we apply the previously suggested changes to attributes, we need also to update this because now attributes contains

 data-geostory-interaction-type and data-geostory-interaction-params